templates/youtube.hbs

```diff
@@ -9,7 +9,19 @@
 
   <div class="component__widget youtube__widget">
 
-    <iframe width="640px" height="360px" style="width:100%;" src="{{_media._source}}?&autoplay={{#if _media._autoplay}}1{{else}}0{{/if}}&rel={{#if _media._showRelated}}1{{else}}0{{/if}}&loop={{#if _loop}}1{{else}}0{{/if}}&modestbranding={{#if _media._modestBranding}}1{{else}}0{{/if}}&color={{#if _media._progressColor}}{{_media._progressColor}}{{/if}}&controls={{#if _media._controls}}1{{else}}0{{/if}}&enablejsapi=1&playsinline=1&iv_load_policy={{#if _media._showAnnotations}}1{{else}}3{{/if}}" frameborder="0" {{#if _media._allowFullscreen}}allowfullscreen{{/if}}></iframe>
+    <iframe width="640px" height="360px" style="width:100%;"
```

Comment:
Suggest wrapping this in an `{{#if _media._source}}`.

At present when it's set to `""`/`undefined` (e.g. when you add the component in the authoring tool and don't edit the form), I get the course embedded within the component(!).

Reply:
This is so cool I'm tempted to ignore your suggestion and leave it as-is 

![image](https://user-images.githubusercontent.com/1249597/105981371-e7c47800-608d-11eb-97fb-acd85e3efba2.png)

Reply:
Slight problem of wrapping the entire `<iframe>` is that the code will error out before `_isReady` is set (infinite loading).

Could instead wrap only the `src` attribute, or do some defensive coding in the JS (might be more time-consuming for you).

Reply:
cheers for the heads-up, I'll have a a think about how best to handle this

Reply:
How about something like this?
![image](https://user-images.githubusercontent.com/1249597/106004888-932ff580-60ab-11eb-9425-57abdbc3fc91.png)